frontend/src/app/main/ui/workspace/sidebar/options/rows/color_row.cljs

```diff
@@ -207,88 +323,95 @@
+                                                     :library-name-wrapper true)
+                                :handle-click-color handle-click-color
+                                :color color}
+        [:*
+         [:div {:class (stl/css :color-name)
+                :title (str color-name)}
+          (str color-name)]
+         [:> icon-button*
+          {:variant "ghost"
+           :class (stl/css :detach-btn)
+           :aria-label (tr "settings.detach")
+           :on-click detach-value
+           :icon i/detach}]]]
+
+       gradient-color?
+       [:> color-info-wrapper* {:class (stl/css-case :color-name-wrapper true
+                                                     :no-opacity disable-opacity
+                                                     :gradient-name-wrapper true)
+                                :handle-click-color handle-click-color
+                                :color color
+                                :opacity true
+                                :select-on-focus select-on-focus
+                                :on-focus on-focus'
+                                :on-blur on-blur'
+                                :on-opacity-change on-opacity-change}
+        [:div {:class (stl/css :color-name)}
+         (uc/gradient-type->string (dm/get-in color [:gradient :type]))]]
+
+       image-color?
+       [:> color-info-wrapper* {:class (stl/css-case :color-name-wrapper true
+                                                     :no-opacity disable-opacity)
+                                :handle-click-color handle-click-color
+                                :color color
+                                :opacity true
+                                :select-on-focus select-on-focus
+                                :on-focus on-focus'
+                                :on-blur on-blur'
+                                :on-opacity-change on-opacity-change}
+        [:div {:class (stl/css :color-name)}
+         (tr "media.image")]]
+
+       :else
+       [:> color-info-wrapper* {:class (stl/css-case :color-name-wrapper true
+                                                     :no-opacity (or disable-opacity
+                                                                     has-multiple-colors)
+                                                     :editing is-editing-text)
+                                :handle-click-color handle-click-color
+                                :color color
+                                :opacity true
+                                :select-on-focus select-on-focus
+                                :on-focus on-focus'
+                                :on-blur on-blur'
+                                :on-opacity-change on-opacity-change}
+
+        [:span {:class (stl/css :color-input-wrapper)}
+         [:> color-input* {:value (if has-multiple-colors
+                                    ""
+                                    (-> color :color clr/remove-hash))
```

Comment:
This function will be executed on each re-render if it does not have multiple colors. Not sure about the cost of remove-hash memory but, shouldn't be memoized?